django/db/models/query.py

```diff
@@ -1304,7 +1314,7 @@ def __getstate__(self):
         obj_dict = self.__dict__.copy()
         if self.queryset is not None:
             # Prevent the QuerySet from being evaluated
-            obj_dict['queryset'] = self.queryset._clone(
+            obj_dict['queryset'] = self.queryset._pre_next_op(
```

Comment:
I guess we could go with just ._clone() and ._chain(). If we want to allow for "in place" querysets, we could just add `clone` argument to chain. If set to False, it effectively does what ._pre_next_op() does right now.